esquery.js

```diff
@@ -44,7 +44,9 @@ function inPath(node, ancestor, path) {
 function matches(node, selector, ancestry) {
     var path, ancestor, i, l, p;
     if (!selector) { return true; }
+    // istanbul ignore if
```

Comment:
I wouldn't have added the lines if I saw any way to reach them. Sometimes people like to put guards in their code which are not actually reachable, so that they are protected for future uses, so the `istanbul ignore` statements allow for this. But currently, as far as I could determine, those lines simply cannot be reached.

Reply:
@brettz9 They were reachable. See https://github.com/estools/esquery/pull/84/commits/ad97039a720200c595def4381b5f7593a38b0993

The other ones probably are too, but I was too lazy to write more tests right now.

Reply:
Ouch, apologies, I will see if I can dig a little deeper... And thanks...